app/views/step_by_step_pages/show.html.erb

```diff
@@ -18,20 +18,27 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <% unless @step_by_step_page.can_be_published? %>
+    <% if !@step_by_step_page.can_be_published? && !@step_by_step_page.scheduled_for_publishing? %>
       <%= render "govuk_publishing_components/components/inset_text" do %>
         <h2 class="govuk-heading-m">To publish this step by step you need to</h2>
 
-        <% if !@step_by_step_page.steps.any? %>
-          <%= link_to('Add at least one step', new_step_by_step_page_step_path(@step_by_step_page), class: 'govuk-link govuk-link--no-visited-state') %>
-        <% else %>
-          <% unless @step_by_step_page.steps_have_content? %>
-            <%= link_to('Add content to all your steps', '#steps', class: 'govuk-link govuk-link--no-visited-state') %>
-          <% end %>
-
-          <% unless @step_by_step_page.links_checked_since_last_update? %>
-            <%= link_to "Check for broken links", { action: "check_links", step_by_step_page_id: @step_by_step_page.id}, method: :post, class: 'govuk-link govuk-link--no-visited-state' %>
-          <% end %>
+        <ul class="govuk-list">
+          <% if !@step_by_step_page.steps.any? %>
+            <li>
+              <%= link_to('Add at least one step', new_step_by_step_page_step_path(@step_by_step_page), class: 'govuk-link govuk-link--no-visited-state') %>
+            </li>
+          <% else %>
+            <% unless @step_by_step_page.steps_have_content? %>
+              <li>
+                <%= link_to('Add content to all your steps', '#steps', class: 'govuk-link govuk-link--no-visited-state') %>
+              </li>
+            <% end %>
+            <% if @step_by_step_page.should_check_links? && !@step_by_step_page.links_checked_since_last_update? %>
```

Comment:
Can `!@step_by_step_page.links_checked_since_last_update?` be part of the `should_check_links?` check?